### PR TITLE
SSME variants and triprop support

### PR DESCRIPTION
**Pull request by Capkirk123**
Add the various SSME variants tested and proposed by NASA for the SSME, including:
* SSME-35
* SSME-50
* SSME-150
* SSME-50X (with deployable nozzle extension)
* SSME-650
* LRBE
Also add support for ModuleBimodalEngineConfigs to:
* RD-701
* RD-704
* LR129
Finally, update RS-84 and TR-107 mass with data I found during my research

Needs https://github.com/NathanKell/ModularFuelSystem/pull/276

### CODE REVIEW

File: GameData/RealismOverhaul/Engine_Configs/LR129_Config.cfg [hunk 1]
```diff
@@ -223,6 +269,19 @@
...
+				
+				atmosphereCurve
+				{
+					key = 0 463
+					key = 1 200
+				}
+			}
 		}
 	}
 }
```
Comment (DRVeyl) [inline GameData/RealismOverhaul/Engine_Configs/LR129_Config.cfg:287]: All TESTFLIGHT patches below need to update for the new MODULE name

File: GameData/RealismOverhaul/Engine_Configs/RD704.cfg [hunk 2]
```diff
@@ -76,15 +76,22 @@
 
 	MODULE
 	{
-		name = ModuleHybridEngine
+		name = ModuleBimodalEngineConfigs
```
[the remark this answers is not in the document]
Comment (Capkirk123) [reply GameData/RealismOverhaul/Engine_Configs/RD704.cfg:79]: The RD-701 and RD-704 do not have TESTFLIGHT configs at the moment

File: GameData/RealismOverhaul/Engine_Configs/RD701.cfg [hunk 3]
```diff
@@ -75,15 +75,22 @@
 
 	MODULE
 	{
-		name = ModuleHybridEngine
+		name = ModuleBimodalEngineConfigs
```
[the remark this answers is not in the document]
Comment (Capkirk123) [reply GameData/RealismOverhaul/Engine_Configs/RD701.cfg:78]: The RD-701 and RD-704 do not have TESTFLIGHT configs at the moment

**Review by DRVeyl — APPROVED** [review]
Modulo lacking TF configs for a couple of engines, looks good to me.  The change to the module name is breaking if there is anything using these configs already.  If that's acceptable, then merge.  (Or if we write an UpgradePipeline...)

**Comment by Capkirk123** [comment]
Conflicts resolved

**Pull request merged**